tools/nuget/nuget.vcxproj

```diff
@@ -194,18 +194,11 @@ popd $(OutDir)</Command>
   <ItemGroup>
     <CustomBuild Include="ebpf-for-windows.nuspec.in">
       <FileType>Document</FileType>
-      <Command Condition="'$(Configuration)|$(Platform)'=='Debug|x64'">powershell -NonInteractive -ExecutionPolicy Unrestricted $(SolutionDir)scripts\Set-Version.ps1 -InputFile $(SolutionDir)tools\nuget\ebpf-for-windows.nuspec.in -OutputFile $(OutDir)ebpf-for-windows.nuspec
-NuGet.exe pack $(OutDir)ebpf-for-windows.nuspec -OutputDirectory $(OutDir)</Command>
-      <Command Condition="'$(Configuration)|$(Platform)'=='NativeOnlyDebug|x64'">powershell -NonInteractive -ExecutionPolicy Unrestricted $(SolutionDir)scripts\Set-Version.ps1 -InputFile $(SolutionDir)tools\nuget\ebpf-for-windows.nuspec.in -OutputFile $(OutDir)ebpf-for-windows.nuspec
-NuGet.exe pack $(OutDir)ebpf-for-windows.nuspec -OutputDirectory $(OutDir)</Command>
-      <Outputs Condition="'$(Configuration)|$(Platform)'=='Debug|x64'">eBPF-for-Windows.2022.4.22.nupkg</Outputs>
-      <Outputs Condition="'$(Configuration)|$(Platform)'=='NativeOnlyDebug|x64'">eBPF-for-Windows.2022.4.22.nupkg</Outputs>
-      <Outputs Condition="'$(Configuration)|$(Platform)'=='Release|x64'">eBPF-for-Windows.2022.4.22.nupkg</Outputs>
-      <Outputs Condition="'$(Configuration)|$(Platform)'=='NativeOnlyRelease|x64'">eBPF-for-Windows.2022.4.22.nupkg</Outputs>
-      <Command Condition="'$(Configuration)|$(Platform)'=='Release|x64'">powershell -NonInteractive -ExecutionPolicy Unrestricted $(SolutionDir)scripts\Set-Version.ps1 -InputFile $(SolutionDir)tools\nuget\ebpf-for-windows.nuspec.in -OutputFile $(OutDir)ebpf-for-windows.nuspec
-NuGet.exe pack $(OutDir)ebpf-for-windows.nuspec -OutputDirectory $(OutDir)</Command>
-      <Command Condition="'$(Configuration)|$(Platform)'=='NativeOnlyRelease|x64'">powershell -NonInteractive -ExecutionPolicy Unrestricted $(SolutionDir)scripts\Set-Version.ps1 -InputFile $(SolutionDir)tools\nuget\ebpf-for-windows.nuspec.in -OutputFile $(OutDir)ebpf-for-windows.nuspec
-NuGet.exe pack $(OutDir)ebpf-for-windows.nuspec -OutputDirectory $(OutDir)</Command>
+      <Outputs>eBPF-for-Windows.2022.4.22.nupkg</Outputs>
```

Comment:
Outputs looks incorrect - the name of the file is $(OutDir)ebpf-for-windows.$(Platform).nupkg? possibly a version number is in the name, too.

Reply:
Good point. Let me fix that :-)